include/DataPoint.h

```diff
@@ -59,11 +59,18 @@ class DataPoint
 
 private:
 
+/// precompiler flag for data storage type
```

Comment:
I think the type should be public. It's potentially useful to have access to it.